servarr/README.md

```diff
@@ -50,8 +50,10 @@ Servarr complete Helm Chart for Kubernetes
 
 | Key | Type | Default | Description |
 |-----|------|---------|-------------|
+| dash.countryCode | string | IT | Inser the Jellyfin country code |
 | dash.mail | string | No default value | Insert Jellyfin login mail (will be used also for Jellyseerr integration) |
 | dash.password | string | No default value | Insert Jellyfin password (will be used also for Jellyseerr) |
+| dash.preferredLanguage | string | it | Inser the Jellyfin preferred language |
```

Comment:
There is a typo (`Inser`), can we fix this? 😋

Reply:
Done 😄